app-component.html

```diff
@@ -14,15 +14,7 @@
 
     <div ng-if="!$ctrl.route.changing && !$ctrl.route.on">
       <div ng-if="$ctrl.path === '/'">
```

Comment:
If we're removing this, can we get rid of (or comment out) the `ng-if` above as well?